Sources/ProtobufferCoding/Decoder/KeyedProtoDecodingContainer.swift

```diff
@@ -93,7 +93,15 @@ class KeyedProtoDecodingContainer<Key: CodingKey>: InternalProtoDecodingContaine
     }
 
     func decode(_ type: Int.Type, forKey key: Key) throws -> Int {
-        throw ProtoError.decodingError("Int not supported, use Int32 or Int64")
+        let keyValue = try extractIntValue(from: key)
```

Comment:
I know that I am a bit late to the party here but given the topic of the PR I was a bit confused about the name of this function. First I was expecting that it actually extracting an int value but it extracts a key that is an integer (as defined by the protobuf spec to uniquely identify fields in a message). Maybe renaming the function and clarifying the functionality in the comment of the function would be beneficial for a first time reader of the code. E.g. `convertKeyToProtobufIntIndex`, `extractProtocolbufferIntKey`,  `extractProtobufIntIndex` or something similar?

Reply:
I agree with you that `extractIntValue` is definitely not the best naming for this function 😁
Changed it to `convertToProtobufferFieldNumber`.